src/Listener/ViewListener.php

```diff
@@ -296,13 +314,36 @@ protected function _getControllerActions()
         $table = $entity = [];
 
         $actions = $this->_crud()->config('actions');
+        $blacklist = (array)$this->_action()->config('scaffold.actions_blacklist');
```

Comment:
I feel this is the wrong way round, inkeeping with our conversation on irc.

It should be possible to define the specific actions to use, and if there is no config (if it's not set, not if it's empty) to use the logic here. So, it should be more along the lines of:

    $actions = $this->_action()->config('scaffold.actions');
    if ($actions !== null) {
        return $actions;
    }
    
    $actions = $this->_crud()->config('actions');
    $blacklist = (array)$this->_action()->config('scaffold.actions_blacklist');
    ...

If it's only possible to define a blacklist - that'll get annoying if an app has an ever increasing number of actions and they aren't wanted to be used as bulk-actions.